src/lib/PetitioRequest.ts

```diff
@@ -251,6 +251,7 @@ export class PetitioRequest {
 	}
 
 	/**
+	 * Will throw HeadersTimeoutError, BodyTimeoutError or a ConnectTimeoutError if the timeout is reached once the request is sent.
```

Comment:
This should be done using the [`@throws` TSDoc tags](https://tsdoc.org/pages/tags/throws) if possible, to describe the situations in which each are thrown and add links to their respective documentation.

Reply:
I thought `@throws` was used when a function throws, but technically when it does throw a timeout it's not from timeout right?

> Used to document an exception type that may be thrown by a **function or property.**